assets/scss/checkout.scss

```diff
@@ -1,3 +1,85 @@
 // =============================================================================
 // Stencil Checkout  - Customize the checkout experience
 // =============================================================================
+
+// Main content
+.ng-checkout-container {
```

Comment:
There are not other classes to use for this container.
```
<div class="ng-checkout-container" id="micro-app-ng-checkout">
  <div class="ng-checkout-view" data-ui-view></div>
</div>
```
a class can be added specifically for this purpose